sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -3,7 +3,9 @@
 ## 1.0.0-preview.4 (Unreleased)
 
 ### Breaking changes
+
```

Comment:
should we add in the new features the information about the new sample?

Reply:
I agree that there's value in letting customers know about the new sample, since it enhances the user experience. We had guidelines for writing changelogs, hadn't we? I'll take a look.

Reply:
Realized we can added it to new features, since we're introducing `FormField{T}` back.
```
`FormField<T>` class added for third party extensibility. See [this sample] for more details.
```